java/arcs/sdk/jvm/JvmEntity.kt

```diff
@@ -19,4 +19,6 @@ interface JvmEntity : Entity {
     fun serialize(): RawEntity
 }
 
-interface JvmEntitySpec<T : Entity> : EntitySpec<T>
+interface JvmEntitySpec<T : Entity> : EntitySpec<T> {
```

Comment:
Why is this JVM only?

Reply:
This should have been in my list above as well. I was avoiding some dependency issues (`arcs.core.data` isn't available for `sdk-js`). I can move this back to sdk `Entity` and use the `Map<String, Any?>` parameter approach that your WIP is using.